tests/operations.go

```diff
@@ -225,6 +230,19 @@ func (ops *Operations) GetCstorVolumeCountEventually(namespace, lselector string
 		60, 10).Should(Equal(expectedCVCount))
 }
 
+// GetCstorVolumeReplicaCountEventually gives the count of cstorvolume based on
+// selecter eventually
+func (ops *Operations) GetCstorVolumeReplicaCountEventually(namespace, lselector string, expectedCVRCount int) bool {
```

Comment:
function names should match
if we are using `GetCstorVolumeReplicaCountEventually` then we should have `GetCstorVolumeReplicaCount`